Mathlib/Data/ENNReal/Inv.lean

```diff
@@ -731,6 +801,27 @@ lemma inv_iSup (f : ι → ℝ≥0∞) : (⨆ i, f i)⁻¹ = ⨅ i, (f i)⁻¹ :
 lemma inv_sInf (s : Set ℝ≥0∞) : (sInf s)⁻¹ = ⨆ a ∈ s, a⁻¹ := by simp [sInf_eq_iInf, inv_iInf]
 lemma inv_sSup (s : Set ℝ≥0∞) : (sSup s)⁻¹ = ⨅ a ∈ s, a⁻¹ := by simp [sSup_eq_iSup, inv_iSup]
 
+lemma mul_iInf_le_iInf_mul {ι : Type*} (u v : ι → ℝ≥0∞) :
+    (⨅ x, u x) * (⨅ x, v x) ≤ ⨅ x, (u * v) x := by
+  refine mul_le_of_forall_mul_le fun a a_u b b_v ↦ ?_
+  rw [lt_iInf_iff] at a_u b_v
+  rcases a_u with ⟨c, a_c, c_u⟩
+  rcases b_v with ⟨d, b_d, d_v⟩
+  simp only [Pi.add_apply, le_iInf_iff]
+  exact fun x ↦ mul_le_mul (a_c.trans_le (c_u x)).le (b_d.trans_le (d_v x)).le (zero_le b)
+    (zero_le u x)
```

Comment:
This proof looks way too complicated
```suggestion
lemma iInf_mul_iInf_le {ι : Type*} (u v : ι → ℝ≥0∞) :
    (⨅ i, u i) * ⨅ i, v i ≤ ⨅ i, u i * v i := iInf_le fun i ↦ mul_le_mul' (le_iInf _) (le_iInf _) 
```
In fact, you could have figured it out from the proof of `iInf_mul_iInf`

Reply:
Well, if that isn't an oversight... thanks for noticing that.

I've kept the name of the lemma (I didn't see other simplifications, and I like the symmetry `iInf_mul_le_mul_iInf `/`iSup_mul_le_mul_iSup`), but if you think it's better, I will implement your version.